src/tmpfiles/tmpfiles.c

```diff
@@ -3482,7 +3483,10 @@ static int parse_line(
                         u = user;
 
                 r = find_uid(u, &i.uid, uid_cache);
-                if (r < 0) {
+                if (r == -ESRCH && arg_graceful) {
+                        log_syntax(NULL, LOG_DEBUG, fname, line, r, "Failed to resolve user '%s', ignoring line: %m", u);
+                        return 0;
```

Comment:
hmm, I think for the /dev/ usecase this is probably not the right approach: I think we should actually accept the line but treat the user as if "-" was supplied if we can't resolve it, i.e. the user was left unspcified.

i.e. there might be device nodes we actually need during early boot, but whose final uid/gid we cannot determine yet. That way the second tmpfiles invocation will adjust the perms and we are good.

(btw, regarding that discussion the other time: we need tmpfiles to create device nodes to implement kmod-auto-loading: in that case we have "ghost" device nodes that exist before the backing kmod is actually loaded)

Reply:
> I think we should actually accept the line but treat the user as if "-" was supplied if we can't resolve it, i.e. the user was left unspcified.

The problem is about permissions. If we have some mode, it might give more permissions to users. For example, if the file is setgid, and instead of being set to some unimportant group, it is now setuid to root. We don't have any configuration like this, but there might be external configuration. Thus if we do this, we'd need some way to figure out what to drop in the permission mask.

Reply:
OK, I like the idea of creating a flag file under `/run/systemd/`. I'll try to implement that and see how it goes.